src/styles/block/stars.css

```diff
@@ -0,0 +1,32 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 > .stars__star:nth-child(1) {
```

Comment:
The style for this selector is the same as the following ones. They can be grouped into a single style block.